guide/private-key-management/case-studies.md

```diff
@@ -0,0 +1,146 @@
+If users have no prior bitcoin knowledge we should expect to spend a significant effort educating them to put them in a position to safely operate the wallet product.
+
+A 2-of-3 [multi-key setup]({{ '/guide/private-key-management/single-user-schemes/#multi-key' | relative_url }}) would seem the most appropriate here, although it will be a significant hurdle in onboarding. Other schemes could be considered but come with distinct downsides for amounts of value we can expect users to store in this use-case. A single-key scheme with an automatic cloud backup, recovery-phrase or single signing device could work at the lower end of the value scale, but start to look like less responsible recommendations with higher values due to their single points of failure.
+
+A key question is the combination of key-storage devices and their distribution. We have many options here, and it might come down to the experience of the target audience and their expected access to the necessary hardware; 
+
+- As low friction as possible (co-managed, no purpose-built signing device)
+- Middle ground (co-managed, one purpose-built signing device)
+- Full sovereignty (one or more purpose-built signing devices) 
+- All keys off-line (two or more purpose-built signing devices)
+
+For this case study we will go with the middle ground option which will require one purpose-built signing device such as a hardware wallet. The other two keys will be one created on the user’s main mobile device and automatically backed up to their cloud provider, and another key held by the wallet-application provider on a server for recovery. Neither of the two keys in the user’s control (mobile and signing device) will require recovery-phrase backups, although this could be offered as an option. 
+
+The idea here is that the user will control the wallet through an app on their main mobile device, but when they are sending funds (paying bills etc) they will need to confirm the transaction on the purpose-built signing device.
+
+Should they lose either their main mobile device, or the purpose-built signing device they can replace the lost key (rotate in a new key) with the help of the recovery key. However, if they lose both the mobile and signing device they will not be able to recover their funds, unless they had also backed up either of the respective recovery-phrases. 
+
+<iframe style="border: 0px solid rgba(0, 0, 0, 0.1);" width="450" height="1000" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fproto%2FSRWlaxbDulsacpPQn2TTri%2FCase-study-prototypes%3Fnode-id%3D6%253A11%26viewport%3D479%252C82%252C0.5%26scaling%3Dscale-down" allowfullscreen></iframe>
+
+#### Design considerations
+- Suitable for monthly transactions
+- Lots of edge cases and infrequent but important situations
+- Key setup, rotation, recovery and signing flows
+
+#### Technical considerations
+- Pitfalls of using hardware wallets in multisig
```

Comment:
Do we have info on what those pitfalls are?

Reply:
It was a reference (although not linked) to this article: https://medium.com/shiftcrypto/the-pitfalls-of-multisig-when-using-hardware-wallets-9b0e98e4c19c

But the real solution here is to write a comprehensive guide to what is required to backup multi-sigs, I haven't yet found one. I'll rephrase to unblock for now.